discord/emoji.py

```diff
@@ -243,7 +243,7 @@ def guild(self):
         name: str
             The new emoji name.
         roles: Optional[list[int]]
-            IDs of the roles for which this emoji is to be whitelisted.  Leave empty to make it unrestricted.
+            List of Roles for which this emoji is to be whitelisted.  Leave empty to make it unrestricted.
```

Comment:
You probably want to reference the `Role` class here, and preface it with "A". I don't like the wording in general.

i.e.

```A :class:`list` or :class:`Roles` that can use this emoji. Leave empty to make it available to everyone.```